assets/js/search-utilities.js

```diff
@@ -12,7 +13,7 @@ function inputFocus(e) {
   if (e.ctrlKey && e.key === '/') {
     e.preventDefault();
     search.focus();
-  // eslint-disable-next-line indent
+    // eslint-disable-next-line indent
```

Comment:
Why do we even need to disable this? Can't we just format the code?

Reply:
:D I don't know why this comment was there at all... Removed it because there is no problem with the formatting ;ddd